bench/irmin-pack/dune

```diff
@@ -27,13 +27,14 @@
  (modules trace_common trace_definitions trace_collection)
  (preprocess
   (pps ppx_repr ppx_deriving.enum))
- (libraries irmin irmin-pack unix lwt repr ppx_repr mtime mtime.clock.os))
+ (libraries irmin irmin-pack unix lwt repr ppx_repr bentov mtime printbox
```

Comment:
you are not using `printbox` in this PR, no?

Reply:
👍 I'll move this out of the PR